gammapy/spectrum/spectrum_extraction.py

```diff
@@ -303,8 +299,10 @@ def write_total_onlist(self):
 
         # Remove column TELMASK since raises an error when stacking it
         # see https://github.com/gammasky/hess-host-analyses/issues/34
+
         for l in on_lists:
-            l.remove_column('TELMASK')
+            if (l.colnames == 'TELMASK'):
```

Comment:
are you sure that this works? I assume you added this because you had an observation table without the column TELMASK? In this case I would write
```
for l in on_lists:
   if 'TELMASK' in l.colnames:
       l.remove_column('TELMASK')

```